pkg/storage/mysql/mysqlstore.go

```diff
@@ -217,7 +217,72 @@ type sqlDBResultsIterator struct {
 }
 
 func (s *sqlDBStore) CreateIndex(createIndexRequest storage.CreateIndexRequest) error {
-	// TODO: Issue-36 Implement Create Index
+	// get all the created indexes
+	indexes, err := s.getIndexes()
+	if err != nil {
+		return fmt.Errorf("failed to get indexes %s", err)
+	}
+	// if an index exits, drop it as sql throws duplicate key_name error
+	err = s.dropExistingIndex(indexes, createIndexRequest)
+	if err != nil {
+		return fmt.Errorf("failed to drop an existing index %s", err)
+	}
+	// create an index
+	createIndexStmt := "CREATE INDEX " + createIndexRequest.IndexName + " ON " +
+		createIndexRequest.IndexStorageLocation + " (" + createIndexRequest.WhatToIndex + ")"
+
+	_, err = s.db.Exec(createIndexStmt)
+	if err != nil {
+		return fmt.Errorf("failed to create index %w", err)
+	}
+
+	return nil
+}
+
+func (s *sqlDBStore) getIndexes() ([]string, error) {
+	var index string
+
+	var indexes []string
```

Comment:
This might be a personal preference, but I usually declare variables just before they're about to be used so it's clearer what they're for